tests/BUILD.bazel

```diff
@@ -3,10 +3,10 @@ sh_test(
     srcs = ["hello_world.test.sh"],
     data = [
         "hello_world.js",
-        "@deno1_14_toolchains//:resolved_toolchain",
+        "@deno_toolchains//:resolved_toolchain",
     ],
     env = {
         "DENO": "$(DENO_PATH)",
     },
-    toolchains = ["@deno1_14_toolchains//:resolved_toolchain"],
+    toolchains = ["@deno_toolchains//:resolved_toolchain"],
```

Comment:
nit: could you not rename this? it makes the delta bigger, and also by having a non-default name for our own local deno toolchain we get some assurance that we haven't accidentally hardcoded the value anywhere.

Reply:
Having a separate name locally makes a lot of sense, but the old name references the now-incorrect version number. I changed it to `deno_local`, but let me know if it should be `deno1_14` specifically.

Reply:
Reverted this back to version 1.14.2 since local tooling should be version agnostic.